server/GameController.js

```diff
@@ -47,6 +47,9 @@ module.exports = function(data) {
 	var startTimer = function(callback) {
 
 		count = 60;
+		if (GameState === POSSIBLE_GAMESTATES.VOTING) {
```

Comment:
maybe comment this for clarity? otherwise seems like a bit of a magic number lol :)